.github/workflows/data_update_v2.yml

```diff
@@ -59,22 +59,15 @@ jobs:
       run: |
         echo "::set-env name=SCRAPER_TIME::$(date)"
         cd ${GITHUB_WORKSPACE}/scraper
+
         OUT_PATH="${GITHUB_WORKSPACE}/site/data/data.v2.json"
-        
-        # Ideally, we'd just run the scraper against all counties like below:
-        #     python scraper_data.py > "${OUT_PATH}"
-        # But we don't want to stop if there are errors, so instead we run each
-        # scraper individually. Then we combine all the outputs and overlay
-        # them on top of the previoius output.
-        (python scraper_data.py alameda || echo '{}') > alameda.json
-        (python scraper_data.py san_francisco || echo '{}') > san_francisco.json
-        (python scraper_data.py solano || echo '{}') > solano.json
-        (python scraper_data.py sonoma || echo '{}') > sonoma.json
-        # Combine outputs with the previous run's saved output:
-        jq -s \
-          '.[0] + .[1] + .[2] + .[3] + .[4]' \
-          "${OUT_PATH}" alameda.json san_francisco.json solano.json sonoma.json \
-          > "${OUT_PATH}"
+
+        python scraper_data.py > today.json 2> errors.txt || true
+        jq -s '.[0] + .[1]' $OUT_PATH today.json > $OUT_PATH
+        ERRORS=`cat errors.txt`
+        if [ -n "${ERRORS}" ]; then
+          curl -X POST -H 'Content-type: application/json' --data "{\"text\":\"${ERRORS}\"}' $SLACK_WEBHOOK_URL
```

Comment:
Wasn’t thinking about this last time around, but we are capturing the errors and only ever emitting them via the webhook, so we never see them in the actions output like we currently do and we’ll miss them entirely if the Slack webhook has a problem:

<img width="983" alt="Screen Shot 2020-09-03 at 10 35 21 AM" src="https://user-images.githubusercontent.com/74178/92148355-4cc2bd00-edd1-11ea-84af-1f090412eb79.png">

Maybe also echo here?

```suggestion
          echo "Encountered the following errors while scraping:"
          echo "------------------------------------------------"
          echo "${ERRORS}"
          curl -X POST -H 'Content-type: application/json' --data "{\"text\":\"${ERRORS}\"}' $SLACK_WEBHOOK_URL
```

Reply:
Great idea! I'm taking this suggestion as it is.